README.md

```diff
@@ -86,7 +86,7 @@ You can currently add Okta OIDC iOS through CocoaPods:
    ***React Native >= 0.60***: With React Native 0.60 pods are added to `Podfile` automatically. Run the commands to install dependencies:
    ```
    cd ios
-   pod install
+   pod install --repo-update
```

Comment:
would this command also handle a brand new installation?

Reply:
how about adding commands for both newly installation and version upgrade scenarios?

Reply:
It will install brand new version of iOS dependency.
There's determined [version in file](https://github.com/okta/okta-react-native/blob/f415ad2a38925370f63b6a222af8ea770befff62/OktaSdkBridgeReactNative.podspec#L23). 
`pod install --repo-update` will fetch info about higher versions and will install a newer version based on determined version in podspec.